src/main/java/uk/gov/justice/laa/crime/cfecrime/utils/FullMeansTestOutcomeCalculator.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.justice.laa.crime.cfecrime.utils;
+
+import lombok.extern.slf4j.Slf4j;
+import uk.gov.justice.laa.crime.cfecrime.cma.enums.FullAssessmentResult;
+import uk.gov.justice.laa.crime.cfecrime.cma.enums.MagCourtOutcome;
+import uk.gov.justice.laa.crime.cfecrime.cma.enums.CaseType;
+import uk.gov.justice.laa.crime.cfecrime.cma.enums.MeansTestOutcome;
+import java.util.HashMap;
+import java.util.Map;
+
+@Slf4j
+public class FullMeansTestOutcomeCalculator {
+
+    public static MeansTestOutcome getFullMeansTestOutcome(FullAssessmentResult result, CaseType caseType, MagCourtOutcome magCourtOutcome){
+        log.debug("Get the outcome of the Full Means Test result = {} caseType = {}", result, caseType);
+
+        //Fail data
+        Map<Integer,CaseType> failMaps = new HashMap<Integer, CaseType>();
+        failMaps.put(1, CaseType.COMMITAL);
+        failMaps.put(2, CaseType.SUMMARY_ONLY);
+        failMaps.put(3, CaseType.EITHER_WAY);
```

Comment:
`failMaps` seems an unclear name.

It appears to be a *set* of case types associated with the magistrates court - the mapping from integer seems unused? What about making it type `Set<CaseType>()`?

And whilst this data gets used only for fail results, the convention is to call a variable by what it is, rather that what it is used for. So how about name: `caseTypesHeardInMagistratesCourt`?

Reply:
renamed to caseTypesHeardInMagistratesCourt